src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public enum Color {
+    Red,

```

Comment:
```suggestion
    RED,

```

all values in enum is constant
constant is in UPPER_CASE